src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java

```diff
@@ -102,4 +105,9 @@ public LoginField openLoginField() {
     public boolean isLogOutIconPresent() {
         return logOutIcon.isPresent();
     }
+
+    public BurgerMenu openBurgerMenu() {
+        burgerMenuBtn.click();
```

Comment:
logger

Reply:
unnecessary, because burgerMenuBtn is ExtendedWebElement. click-action will be logged by framework